frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -0,0 +1,86 @@
+import {useQuery} from "@tanstack/react-query";
+import {SkeletonText, Table, TableContainer, Tbody, Td, Th, Thead, Tr} from "@chakra-ui/react";
+import * as InquiriesService from "../../client/services/inquiriesService.ts";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+
+// Dayjs Configurations
+// eslint-disable-next-line
+dayjs.extend(utc);
+// eslint-disable-next-line
+dayjs.extend(timezone);
+// eslint-disable-next-line
+const userTimezone = dayjs.tz.guess();
+
+function getInquiriesQueryOptions() {
+    return {
+        queryKey: ["inquiries"],
+        queryFn: () => InquiriesService.readInquiries(),
+    };
+}
+
+// Format date to user's timezone
+// ex. Sep 17, 2024 14:13 PM
+function formatDate(date: Date): string {
+    try {
+        // eslint-disable-next-line
```

Comment:
I'm curious -- what Codacy error was throwing here, and what's our reason for ignoring? Remember, when we ignore the linter, there should at least be a quick comment mentioning why.

Reply:
All of this was moved from @ilhanbae 's  code so maybe he can answer.

Reply:
Hi @Eugene-Polonsky, I believe it was something related to error catching. 
I tried to undo the eslint-disable here: https://github.com/Incredihire/Survey/pull/54

But actually it doesn't seem to show Codacy errors anymore.

Reply:
![codacyError](https://github.com/user-attachments/assets/4c71317c-98ad-42a2-b50a-2d8dfde98f36)

Reply:
So, I used try catch block to handle errors, but now I see that I should've checked if each variables were valid first.